core/lib/basic_types/src/protocol_version.rs

```diff
@@ -45,8 +45,8 @@ pub enum ProtocolVersionId {
     Version21,
     Version22,
     Version23,
-    Version24,
     Local,
+    Version24,
```

Comment:
is this on purpose?

Reply:
yes! otherwise upgrade unit tests fail. There was a comment about it sometime ago, but apparently it was removed during merge